onnxruntime/contrib_ops/cuda/bert/attention_impl.cu

```diff
@@ -251,7 +260,8 @@ Status QkvToContext(
   T* v = k + elements_k;
 
   // For fused TRT attention, qkv need transpose to BxSxNx3xH
-  bool use_fused_kernel = (nullptr != fused_runner && data.bias != nullptr);
+  bool use_fused_kernel = (nullptr != fused_runner && data.bias != nullptr && !parameters.is_unidirectional);
```

Comment:
>data.bias != nullptr [](http://example.com/codeflow?start=54&length=20)

Why data.bias is necessary to use fused kernel?

Reply:
Bias==nullptr is for quantized attention. It uses transpose instead of add_bias_transpose so it need extra code to handle it. It is possible to support it but not done in this PR.